tests/effective_dps_test.cpp

```diff
@@ -336,8 +336,8 @@ static void check_two_handed_axes( const std::function<Approx( const std::string
         CHECK( calc_expected_dps( "lobotomizer" ) == 24.0 );
         CHECK( calc_expected_dps( "ax" ) == 20.25 );
         CHECK( calc_expected_dps( "copper_ax" ) == 13.75 );
-        CHECK( calc_expected_dps( "e_combatsaw_on" ) == 28.0 );
-        CHECK( calc_expected_dps( "combatsaw_on" ) == 28.0 );
+        CHECK( calc_expected_dps( "e_combatsaw_on" ) == 35.07 );
+        CHECK( calc_expected_dps( "combatsaw_on" ) == 35.07 );
```

Comment:
This is a really big buff, and I'm not sure it's correct.

Reply:
The only thing that changed there was their chance to hit improved when they are turned on, on the basis of them fitting into the "every" category for surface.

Reply:
I'm not sure chainsaws should be `every`, but I don't know enough about chainsaws to make that judgement. I would definitely drop their damage to keep them about where they started if necessary.

Reply:
> I'm not sure chainsaws should be `every`, but I don't know enough about chainsaws to make that judgement. I would definitely drop their damage to keep them about where they started if necessary.

I wasn't sure about that either, but the balance document lists "Jagged tearing weapons" as having "every" property and I wanted to try and follow that to the best of my ability.